testsuite/model/src/test/java/org/keycloak/testsuite/model/session/UserSessionPersisterProviderTest.java

```diff
@@ -594,8 +594,8 @@ public static void assertSession(UserSessionModel session, UserModel user, Strin
         assertEquals(user.getUsername(), session.getLoginUsername());
         assertEquals("form", session.getAuthMethod());
         assertTrue(session.isRememberMe());
-        assertTrue(session.getStarted() >= started - 1 && session.getStarted() <= started + 1);
-        assertTrue(session.getLastSessionRefresh() >= lastRefresh - 1 && session.getLastSessionRefresh() <= lastRefresh + 1);
+        assertTrue(session.getStarted() >= started - 2 && session.getStarted() <= started + 2);
+        assertTrue(session.getLastSessionRefresh() >= lastRefresh - 2 && session.getLastSessionRefresh() <= lastRefresh + 2);
```

Comment:
Why is this change necessary?

Reply:
It started to fail after I changed `userSessionprovider.removeLocalUserSessions()` to `reinitializeKeycloakSessionFactory` in `createSessionsInPersisterOnly()`. 
At the end I decided to fix it more nicely by capturing `started` directly from the created offline session rather than from `Time.currentTime()` after the `createSessionsInPersisterOnly() is called`.